single_tech_samples/datafactory/tests/integrationtests/tests/test_pipeline_ingest_to_bronze.py

```diff
@@ -24,5 +24,5 @@ def test_pipeline_succeeded(adf_pipeline_run, blob_service_client):
     storage_stream_downloader = blob_container_client.download_blob(blob.name, None, None)
     blob_bytes = storage_stream_downloader.readall()
     
-    contents_array = blob_bytes.decode("UTF-8").split("\r\n")
+    contents_array = blob_bytes.decode("UTF-8").split("\n")
```

Comment:
I've tested locally it works OK with splitting '\r\n', you met issue with this config so that have to remove '\r'? Just curious :-D

Reply:
As we quickly discussed we use \n instead of \r\n